Sources/UI/Chat/Cells/MessageTableViewCell.swift

```diff
@@ -348,6 +342,26 @@ open class MessageTableViewCell: UITableViewCell, Reusable {
         }
     }
     
+    private func setupReplyButton(_ button: UIButton) {
```

Comment:
nit: `setUp` vs. `setup` -> https://grammarist.com/spelling/set-up-vs-setup/

Reply:
I understand your concern. We use `setup` everywhere and it's just usual in code. It looks weird when I see `setUp`.
@b-onc what do you think?

Reply:
Although incorrect, I'm also used to using `setup` as `setUp`, I'd say we keep it as we understand

Reply:
I don't think grammar is a question of one's preference. `setupButton`, `buttonSetup` and `setUpButton` are different things. It was just a nit comment to raise awareness of this for the future.